src/Exception/ApiException.php

```diff
@@ -50,7 +50,7 @@ public function __construct(GuzzleRequestException $previous, $message = '')
 
         if ($this->previous->hasResponse()) {
             $this->response = $this->previous->getResponse();
-            $this->requestId = $this->response->getHeader('X-Contentful-Request-Id')[0];
+            $this->requestId = $this->response->getHeaderLine('X-Contentful-Request-Id');
```

Comment:
The coverage tools strips out all headers by default, so this raised a warning because that header is not available.

PSR-7 assumes that headers can have multiple values, and for some it _can_ be true (like set-cookie), but the vast majority is basically a key => value scenario. Because of this, `getHeaderLine` is simply a shortcut instead of doing something like
``` php
$requestIdHeader = $this->response->getHeader('X-Contentful-Request-Id');
$this->requestId = $requestIdHeader ? $requestIdHeader[0] : '';
```

`X-Contentful-Request-Id` will only ever be a single line, so there's really no problem using `getHeaderLine`, and on the plus side we don't have to make an explicit check ourselves.